modules/rosa-sre-access-privatelink-vpc.adoc

```diff
@@ -0,0 +1,13 @@
+[id="rosa-sre-access-privatelink-vpc.adoc_{context}"]
+= SRE access through PrivateLink VPC endpoint service 
+
+PrivateLink VPC endpoint service is created as part of the ROSA cluster creation. 
+
+When you have a PrivateLink ROSA cluster, its Kubernetes API Server is exposed through a load balancer that can only be accessed from within the VPC by default. Red Hat site reliability engineering (SRE) can connect to this load balancer through a VPC Endpoint Service that has an associated VPC Endpoint in a Red Hat owned AWS account. This endpoint service contains the name of the cluster, which is also in the ARN.
+
+Under the *Allow principals* tab, a Red Hat owned AWS account is listed. This specific user ensures that other entities cannot create VPC Endpoint connections to the PrivateLink cluster’s Kubernetes API Server. 
+
+When Red Hat SREs access the API, this fleet management plane can connect to the internal API through the VPC endpoint service.
```

Comment:
"This fleet management plane" seems confusing. I didn't see the term "fleet management plane in this document before this. Maybe just "The fleet..."

Reply:
Agreed, but that was specific language provided by SRE, so I'm going to keep it as is